BuyPolish/PolaUITests/PageObjects/ScanBarcodePage.swift

```diff
@@ -11,4 +11,9 @@ class ScanBarcodePage: BasePage {
         _ = app.staticTexts["OCEŃ NAS"].waitForExistence(timeout: waitForExistanceTimeout)
         return InformationPage(openFrom: self)
     }
+
+    func isReviewRequestVisible() -> Bool {
+             let element = app.scrollViews.otherElements.containing(.staticText, identifier: "Podoba Ci się aplikacja „Pola”?").element
+             return element.waitForExistence(timeout: 2)
```

Comment:
Zamiast `2` użyj `waitForExistanceTimeout` jest to zmienna w `BasePage`. Dzięki temu wszystkie timeouty mają tą samą wielkość i w przypadku, gdy będzie potrzeba zmiany zrobi się to globalnie.